src/Add.js

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Link } from "react-router-dom";
+
+export default function Add(props) {
+  const newDate = useRef(new Date());
+
+  const [currentDay, setCurrentDay] = useState(newDate.current.getDate());
+  const [currentMonth, setCurrentMonth] = useState(newDate.current.getMonth());
+  const [currentYear, setCurrentYear] = useState(newDate.current.getFullYear());
+  const [currentHours, setCurrentHours] = useState(newDate.current.getHours());
+  const [currentMinutes, setCurrentMinutes] = useState(
+    newDate.current.getMinutes()
+  );
+  setInterval(() => {
+    newDate.current = new Date();
+    setCurrentDay(newDate.current.getDay());
+    setCurrentMonth(newDate.current.getMonth());
+    setCurrentYear(newDate.current.getYear());
+    setCurrentHours(newDate.current.getHours());
+    setCurrentMinutes(newDate.current.getMinutes());
+  }, 10000000);
+
+  const dayIndex = new Date().getDay();
+  const getDayName = (dayIndex) => {
+    const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+    return days[dayIndex];
+  };
+
+  const monthIndex = new Date().getDay();
+  const getMonthName = (dayIndex) => {
+    const month = [
+      "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec",];
+    return month[monthIndex];
+  };
+
+  const dayName = getDayName(dayIndex);
+  const monthName = getMonthName(dayIndex);
+
+  const day = `${currentDay}`;
+  const month = `${currentMonth + 1}`;
+  const year = `${currentYear}`;
+  const minutes = `${currentMinutes}`;
+  const hours = `${currentHours}`;
+
+  useEffect(() => {
+    console.log(day);
+    console.log(month);
+    console.log(year);
+    console.log(hours);
+    console.log(minutes);
+  });
+
+  console.log(props.id);
+
+  const [list, setList] = useState([]);
+  const [input, setInput] = useState("");
+
+  const addTodo = (todo) => {
+    const newTodo = {
+      id: Math.random(),
+      todo: todo,
+    };
+
+    setList([...list, newTodo]);
+    setInput("");
+  }
+
+  const deleteTodo = (id) => {
+    const newList = list.filter((todo) => todo.id !== id);
+    setList(newList);
+  };
+
+  const przekresl = event => {

```

Comment:
Never write names in other language than english. `const markAsDone = () =>`  and don't style through style property :D This why we are using react for. Add new field to a todo item `done: boolean;` and based on this value show different styles. Also try to implement everything with styled components https://styled-components.com/